discovery-provider/src/app.py

```diff
@@ -269,6 +269,9 @@ def default(self, o):
             if "url" in test_config["db"]:
                 database_url = test_config["db"]["url"]
 
+    if shared_config["discprov"]["hostname"]:
+        app.config["SERVER_NAME"] = shared_config["discprov"]["hostname"]
```

Comment:
do you foresee any problems overwriting this value since this is a flask default?

Reply:
There is no default value for `SERVER_NAME`, so i think it should be fine